### ISSUE TITLE
question about LINKSET usage: property should be array of RIDs

### ISSUE THREAD

**Issue by rdelangh** [orientechnologies/orientdb#7194]
## Expected behavior  
I don't entirely understand how to use LINKSET properties in a class:

In my class "parents", I defined a property "childs"  of type LINKSET.
```
CREATE property parents.childs LINKSET (MANDATORY TRUE, NOTNULL FALSE)
```

I then create (using INSERT) a document of type "parents", with property "childs" equal to NULL. The RID of this "parents" document is #354:0

Then I want to add a RID of some document elsewhere, into this LINKSET property "childs":
```
update #354:0 ADD childs = #359:0
```

but when I read the value for this "childs" property, it remains NULL:
```
SELECT childs FROM #354:0
{
    "result": [
        {
            "@type": "d",
            "@rid": "#-2:0",
            "@version": 0
        }
    ],
    "notification": "Query executed in 0.042 sec. Returned 1 record(s)"
}
```

What am I doing wrong?

**Comment by luigidellaquila** [comment]
Hi @rdelangh 

I think it's a bug, I'll check it.
In the meantime, I suggest you to initialize it with an empty array, eg.

```
insert into parents set childs = []
```

Thanks

Luigi

**Comment by rdelangh** [comment]
@luigidellaquila 
indeed, initializing this LINKSET property first with an empty array, then doing the ADD, works fine.
I will use this method, but also about this topic there was no documentation. Can a clear set of steps about how LINKSETs must be initialized, then used, be added in the documentation of ODB?

**Comment by luigidellaquila** [comment]
Hi @rdelangh 

I just pushed a fix on branch 2.2.x, the fix will be released with 2.2.20

Thanks

Luigi
